charts/orchestrator/values-int.yaml

```diff
@@ -0,0 +1,50 @@
+#################################################################################
+# Copyright (c) 2022,2023 T-Systems International GmbH
+# Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+################################################################################
+
+ingress:
+    # -- If you want to enable or disable the ingress
+  enabled: true
+    # -- a reference to an Ingress Class resource that contains additional configuration including the name of the controller that should implement the class
+  className: nginx
+    # -- Annotations to add to the ingress
+  annotations:
+    nginx.ingress.kubernetes.io/backend-protocol: HTTP
+    cert-manager.io/cluster-issuer: "letsencrypt-prod"
+    nginx.ingress.kubernetes.io/affinity: "cookie"
+    nginx.ingress.kubernetes.io/session-cookie-max-age: "172800"
+
+    # -- Host of the application on which application runs
+  host: "orchestrator.int.cx.dih-cloud.com"
+
+secretRef: "managed-service-orchestrator-int-secret"
+
+postgresql:
+    # -- Enable the dependency postgres database
+  enabled: true
+  metrics:
+    containerSecurityContext:
+      enabled: false
+  auth:
+    username: "orchdbuser"
```

Comment:
I will accept this change, since it at least fixes the default values, but you need to remove the hardcoded passwords. either use a secure vault, or use the auto-generated passwords provided by the bitnami chart